src/custom/components/ArrowWrapperLoader/index.tsx

```diff
@@ -132,8 +126,8 @@ export function ArrowWrapperLoader({ onSwitchTokens, setApprovalSubmitted }: Arr
   }
 
   return (
-    <Wrapper showLoader={showLoader} onClick={handleClick}>
-      <ArrowDownIcon showLoader={showLoader} />
+    <Wrapper $showloader={showLoader ? true : undefined} onClick={handleClick}>
+      <ArrowDownIcon $showloader={showLoader ? true : undefined} />
```

Comment:
Why is the ternary required? The prop should accept a Boolean only and showLoader is a Boolean no?

Reply:
@W3stside That's what I thought too, and how I had it. However, I get this error in console when I just do `$showloader={showloader}`

```
Warning: Received `false` for a non-boolean attribute `$showloader`.

If you want to write it to the DOM, pass a string instead: $showloader="false" or $showloader={value.toString()}.

If you used to conditionally omit it with $showloader={condition && value}, pass $showloader={condition ? value : undefined} instead.
```

Hence, I picked the suggestion to do ` $showloader={condition ? value : undefined}` as we don't want to pass it to the DOM (would have to turn it into a string first then). 

I also tried `$showloader={showLoader && showLoader}` but this returned the same error.

So right now the `$showloader` type is:
```
interface ShowLoaderProp {
  $showloader: boolean | undefined
}
```

Reply:
It's strange I'll try a small fix on top of this

Reply:
The point I think is that the prop is passed down to the html element.
This is what causes the warning/error with unknown prop or whatever it's called, which is not present anymore in this case because I believe it's fixed with this code :)